### ISSUE TITLE
Generated QR is not scanned by wallets. Kotlin Modal QR works at the same time

### ISSUE THREAD

**Issue by Raenar4k** [WalletConnect/web3modal-swift#51]
**Describe the bug**
As title - Kotlin version of Modal works fine, but swift samples QR is not recognized by wallets.
Here are both side by side
<img width="396" alt="image" src="https://github.com/WalletConnect/web3modal-swift/assets/13849790/7292bebb-8b3c-478e-a4d6-3f44b7218627">
<img width="342" alt="image" src="https://github.com/WalletConnect/web3modal-swift/assets/13849790/43edffd8-ab85-4f79-893a-8d6db993b1a0">

**Comment by llbartekll** [comment]
hey @Raenar4k could you check last version?

**Comment by Raenar4k** [comment]
After pulling updates and building - i get this error while running.
So i cannot get to QR part
<img width="771" alt="image" src="https://github.com/WalletConnect/web3modal-swift/assets/13849790/69713c8f-017a-4660-9ac9-d8d1f8637b3e">

**Comment by llbartekll** [comment]
@Raenar4k could you try to upgrade to 1.3.1?

**Comment by Raenar4k** [comment]
@llbartekll 
Now sample works, yes.
It generates this QR, which is not recognized. Im testing on Metamask, which works for kotlin sample
<img width="340" alt="image" src="https://github.com/WalletConnect/web3modal-swift/assets/13849790/9eb5f9f0-b62f-42c4-8d52-8197ce0281bd">

**Comment by llbartekll** [comment]
@Raenar4k could you give me steps to reproduce or make a screen recording?
I'm connecting with MetaMask without any issues

**Comment by Raenar4k** [comment]
@llbartekll 
Sure. Here are 2 emulators side by side - kotlin WC demo and swift WC demo.
Scanned by android device - Pixel 7 pro, by metamask app
https://github.com/WalletConnect/web3modal-swift/assets/13849790/9d0ad9d4-b515-488c-9300-461c71b1f19b

**Comment by llbartekll** [comment]
I just tried to reproduce and I experience no issues when scanning qr code with Kotlin wallet.

**Comment by Raenar4k** [comment]
>with Kotlin wallet.  

I also have no issues when scanning QR from Kotlin wallet.
It is Swift version on iOS emulator that is not recognized @llbartekll

**Comment by llbartekll** [comment]
> It is Swift version on iOS emulator that is not recognized

yes, this is what i mean, swift dapp and any wallet(kotlin and swift) connects. can you try to copy/paste the uri?

**Comment by Raenar4k** [comment]
I used 3rd party QR scanner - reading URI and then clicking on it works - Metamask connects and app shows balance, signing message also works.
Trust wallet also connects, but lists all chains for some reason, when i selected only etherium.

There is an issue with image of QR code generated by dapp.
So i've recorded a little test - where i use link from swift dapp and generate QR code with its contents - and it works instantly.

https://github.com/WalletConnect/web3modal-swift/assets/13849790/d9e22004-6736-44d6-8600-a934a477fddc

**Comment by Raenar4k** [comment]
I used 3rd party QR scanner - reading URI and then clicking on it works - Metamask connects and app shows balance, signing message also works.
Trust wallet also connects, but lists all chains for some reason, when i selected only etherium.

There is an issue with image of QR code generated by dapp.
So i've recorded a little test - where i use link from swift dapp and generate QR code with its contents - and it works instantly.

https://github.com/WalletConnect/web3modal-swift/assets/13849790/00937a20-5b89-4457-8344-25d5e3ab4e2d

**Comment by llbartekll** [comment]
hmm, this is interesting, on ios im able to scan the qr code with any wallet, including MetaMask and all works
Do you experience the issue when you scan the qr code with other wallets? or only metamask?

**Comment by Raenar4k** [comment]
Same issue with TrustWallet.
Did not test on others
